packages/file_selector/file_selector/example/lib/get_directory_page.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:file_selector/file_selector.dart';
+import 'package:flutter/material.dart';
+
+/// Screen that shows an example of openFile(s)
+class GetDirectoryPage extends StatelessWidget {
+  void _getDirectoryPath(BuildContext context) async {
+    final String initialDirectory = '~/Desktop';
+    final String confirmButtonText = 'Choose a cool directory';
```

Comment:
Nit: This is a *lot* of text for a button. Maybe pick something more realistic to demonstrate, like "Choose"?

Reply:
Done!